jest.config.js

```diff
@@ -14,6 +15,7 @@ module.exports = {
     '!jest.config.js',
     '!<rootDir>/server.js',
     '!webpack.common.js',
+    '!**/*.scss',
```

Comment:
@beejay1293 I think this command for jest to ignore scss files is not needed anymore since we have `sass-lint` configured to the base project